rules.neon

```diff
@@ -106,6 +106,10 @@ conditionalTags:
 		phpstan.rules.rule: %strictRules.numericOperandsInArithmeticOperators%
 	PHPStan\Rules\Operators\OperandInArithmeticPreIncrementRule:
 		phpstan.rules.rule: %strictRules.numericOperandsInArithmeticOperators%
+	PHPStan\Rules\Operators\OperandInArithmeticUnaryMinusRule:
+		phpstan.rules.rule: %strictRules.numericOperandsInArithmeticOperators%
```

Comment:
Only for bleeding edge. Please check 1.x to see how it was done (it's an array).